### PR TITLE
Basic player movement

### PR DESCRIPTION
**Pull request by hallur**
Note: No rotation and no animation

### CODE REVIEW

File: Assets/Prefabs/Player.prefab [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+  m_PrefabInternal: {fileID: 100100000}
+  serializedVersion: 5
+  m_Component:
+  - component: {fileID: 4116187538220734}
+  - component: {fileID: 212527402164969680}
+  - component: {fileID: 114142521996636744}
+  - component: {fileID: 50849203600294468}
+  m_Layer: 0
+  m_Name: Player
+  m_TagString: Untagged
```
Comment (dadisigursveinn) [inline Assets/Prefabs/Player.prefab]: player prefab should be tagged

File: Assets/Prefabs/Player.prefab [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+  m_Icon: {fileID: 0}
+  m_NavMeshLayer: 0
+  m_StaticEditorFlags: 0
+  m_IsActive: 1
+--- !u!4 &4116187538220734
+Transform:
```
Comment (dadisigursveinn) [inline Assets/Prefabs/Player.prefab:33]: Prefab should be zeroed in world

File: Assets/Sprites/player.png.meta [hunk 3]
```diff
@@ -0,0 +1,145 @@
+fileFormatVersion: 2
+guid: d361d21869961a94cbe6efb46bbccd83
+timeCreated: 1512387772
+licenseType: Free
+TextureImporter:
+  fileIDToRecycleName:
+    21300000: 1-player-right_foot_forward
```
Comment (dadisigursveinn) [inline Assets/Sprites/player.png.meta:7]: not a fan of this convention.

Needs to be revisited with whole group

**Review by dadisigursveinn — CHANGES_REQUESTED** [review]

File: Assets/Prefabs/Player.prefab (hunk 2 above)
Comment (hallur) [reply Assets/Prefabs/Player.prefab:33]: Fixed

**Review by dadisigursveinn — APPROVED** [review]

File: Assets/Prefabs/Player.prefab (hunk 1 above)
Comment (hallur) [reply Assets/Prefabs/Player.prefab]: Fixed

**Comment by dadisigursveinn** [comment]
[Player Movement](https://trello.com/c/kPGCLzh4/65-player-movement)

**Pull request merged**